docs/theory/enzyme_kinetics.rst

```diff
@@ -3,3 +3,119 @@ Enzyme kinetics
 ===============
 
 This document explains the assumptions about enzyme kinetics that Maud uses.
+
+Modular rate law
+================
+
+This section outlines the assumptions made with the modular rate law, and includes a derivation of 2 substrate (A, B),
+2 product (P, Q) random mechanism with competitive inhibitor I. It also highlights the general structure of the
+modular rate law used in Maud. The modular rate law framework was taken from [1],
+and was adapted to suit our structure.
+
+Assumptions
+-----------
+
+The assumptions used in the modular rate law are listed below:
+    - the metabolite binding occurs in a random order,
+    - binding does not occur simultanesouly,
+    - substrates and products cannot bind at the same time,
+    - metabolite binding rates are much higher than the interconversion of substrate to product (rapid equilibrium assumption),
+    - metabolite binding affinity is independent of order.
+
+
+Example: 2 products and 2 substrate network
+-------------------------------------------
+
+.. figure:: random-bibi.png
+
+    A random mechanism with 2 products and 2 substrates with a slow conversion step. All of the reactant
+    binding/release steps are in rapid equilibrium.
+
+For a random Bi-Bi network with the above assumptions, the rate will be the following:
+
+.. math::
+   v = E_t \frac{kcat_1 a' b' - kcat_2 p' q'}{(1 + a')(1 + b') + (1 + p')(1 + q') -1}
+
+where for metabolite X the corresponding term is given by,
+
+.. math::
+   x' &= \frac{X}{K_m^{x}} \\
+   K_m^{x} &= \frac{[X] \bullet [E_{i-1}]}{[E_i]}
+
+where :math:`E_{i-1}` is the enzyme state not bound to metabolite X, and,
+:math:`E_i` is the enzyme state bound to metabolite X
+
+A derivation is shown below. The rate is determined by the interconversion between substrate to product and using
+elementary mass action kinetics is:
+
+.. math::
+   v = kcat_1 EAB - kcat_2 EPQ
+
+Because of the rapid equilibrium assumption the dissociation constants are assumed 
```

Comment:
Because of the rapid equilibrium assumption, the Michaelis-Menten constants are approximated by the dissociation constants (i.e. in XXX case the ratio of reverse to forward elementary rate constants).